synapse/lib/httpapi.py

```diff
@@ -139,48 +145,6 @@ async def reqAuthAdmin(self):
 
         return True
 
-    async def reqAuthAllowed(self, path):
```

Comment:
I will have downstream code that will need to be updated eventually to handle this removal

Reply:
yep!  figured we'd just break it and fix it :D